docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -52,7 +52,34 @@ This command should result in three PEM-encoded files: `auth.crt`,
 `auth.key`, and `auth.cas` (certificate, private key, and CA certs
 respectively). Later, you will configure the plugin to use these credentials to
 connect to the Auth Service directly.
+</TabItem>
+<TabItem label="Connect to the Proxy or Auth Service" options="Helm Chart">
+The following `tctl auth sign` command impersonates the `access-plugin` user,
+generates signed credentials, and writes an identity file to the local
+directory:
+
+```code
+$ tctl auth sign --user=access-plugin --out=auth
+```
+
+Then create a Kubernetes secret:
+
+```code
+$ kubectl create secret generic teleport-mattermost-identity --from-file=auth_id=auth.pem
+```
 
+Teleport's Access Request plugins listen for new and updated Access Requests by
```

Comment:
Should "Access Requests" be capitalized? Does this refer to ephemeral requests or to a proper noun like the plugins? 

```suggestion
Teleport's Access Request plugins listen for new and updated access requests by
```

Reply:
This is a Teleport product, so I've been asking for it to be capitalized.